### PR TITLE
allow to compile an individual package

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve?

An individual package can't be compiled successfully without compiling the whole workspace.

### Check List 

Tests 

- No code

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

File: fuzz/fuzzer-afl/Cargo.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ publish = false
 
 [features]
 default = ["protobuf-codec"]
-protobuf-codec = ["fuzz-targets/protobuf-codec"]
+protobuf-codec = [ "fuzz-targets/protobuf-codec" ]
```
Comment (sticnarf) [inline fuzz/fuzzer-afl/Cargo.toml:8]: The format looks inconsistent

**Review by gengliqi — APPROVED** [review]

**Review by sticnarf — APPROVED** [review]

**Pull request merged**